tools/cellprofiler/runcp.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+import sys
+import os
+
+DEVNULL = open(os.devnull,'wb') #This is needed to supress ilastik error due to running CellProfiler in headless configuration
+cp_path = os.environ['CP_PATH']
+if(sys.argv > 3):
+	print "Too many command line arguments"
+if(sys.argv[1] == 'bcc'):
+	os.chdir(cp_path+'/bcc/')
+	subprocess.call('python createSymLinks.py',shell=True)
+	subprocess.call('python createFileList.py',shell=True)
```

Comment:
Do these scripts create unique output files/directories everytime they are run? Essentially, we want to be sure that if two users run this tool in parallel, they will not clobber each other's files.